src/main.rs

```diff
@@ -123,51 +113,25 @@ fn main() {
     }
     let key = parts[0];
     let signer = get_signer(key);
-    tracing::info!(key = ?signer.public_key(), "loaded signer");
-    let public_key = signer.public_g1();
-    let (apk, _, _) = bn254::get_points(&[public_key], &[signer.public_key()], &[]).unwrap();
-    let g1_point = convert_to_g1_point(apk).unwrap();
-    println!(
-        "public key G1 coordinates: ({}, {})",
-        g1_point.X, g1_point.Y
-    );
-    println!("key: {}", key);
-    println!("parts: {:?}", parts);
-    println!("me: {:?}", me);
-    // std::process::exit(0);
-
-    // Configure my port
     let port = parts[1].parse::<u16>().expect("Port not well-formed");
     tracing::info!(port, "loaded port");
 
-    // Configure bootstrappers (if provided)
-    let bootstrappers = matches.get_many::<String>("bootstrappers");
-    let mut bootstrapper_identities = Vec::new();
-    if let Some(bootstrappers) = bootstrappers {
-        for bootstrapper in bootstrappers {
-            let parts = bootstrapper.split('@').collect::<Vec<&str>>();
-            let verifier = get_signer(parts[0]).public_key();
-            let bootstrapper_address =
-                SocketAddr::from_str(parts[1]).expect("Bootstrapper address not well-formed");
-            bootstrapper_identities.push((verifier, bootstrapper_address));
-        }
-    }
-
     // Configure network
     const MAX_MESSAGE_SIZE: usize = 1024 * 1024; // 1 MB
-    let p2p_cfg = authenticated::Config::aggressive(
+    let my_addr = SocketAddr::new(IpAddr::V4(Ipv4Addr::UNSPECIFIED), port);
+    let my_local_addr = SocketAddr::new(IpAddr::V4(Ipv4Addr::LOCALHOST), port);
+    let p2p_cfg = lookup::Config::aggressive(
```

Comment:
the docs for `aggressive` advise not to use it for production https://docs.rs/commonware-p2p/latest/commonware_p2p/authenticated/lookup/struct.Config.html#method.aggressive